notebooks/16-explore-other-neynar-datasets.ipynb

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
<p>Do we need to keep this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bleu/farcaster-social-graph/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
just wanted to document the neynar datasets columns. do you think that's relevant? imo I should also put the command to download the dataset here